packages/react-ui-core/src/AutoSuggestField/AutoSuggestField.js

```diff
@@ -166,12 +166,12 @@ export default class AutoSuggestField extends PureComponent {
     }, callback)
   }
 
-  handleSubmit = () => {
-    this.props.onSubmit(this.state.value)
+  handleSubmit = event => {
+    this.props.onSubmit(this.state.value, event.type)
     this.updateValueAndClose(this.state.value)
   }
 
-  handleSuggestionSelection = value => {
+  handleSuggestionSelection = (value, ix, eventType) => {
```

Comment:
Do we need `ix`? It doesn't look like it is being used at all

Reply:
We don't need IX for the most part, but to remain backward compatible it's required

Reply:
Can we come up with a better name? I have no idea what ix means